tests/store.spec.js

```diff
@@ -522,4 +522,25 @@ describe('Store', function () {
     it.todo('fetches a single resource')
     it.todo('updates the resource with the response')
   })
+
+  describe('request', function () {
```

Comment:
this can be moved to a separate PR if needed to reduce size

Reply:
Seems manageable to me. These tests seem good to include.